packages/chain-ethereum-verifiable/src/types.ts

```diff
@@ -0,0 +1,30 @@
+export type EIP712Domain = {
+	name: string
+	version: string
+	chainId: number
+	verifyingContract: string
+	salt: string
+}
+
+export type EIP712VerifiableSessionData = {
+	signature: Uint8Array
+	domain: EIP712Domain
```

Comment:
i think domain (as in document.location, not EIP712 Domain) can just be left out of the signer, it's only in the SIWE signer because the SIWE standard asks for a domain

in metamask, if you ask for a SIWE signature with an invalid/incorrect domain, the SIWE "log in to X website" interface doesn't show up, and it just shows up as text to be signed

Reply:
oh my bad! i mistook the "domain" field in the SIWESigner for the EIP712 domain. yeah this makes sense, i'll removei t

Reply:
ah so this is kind of confusing, but the reason why i included the EIP712Domain in the session data object was so that inside verifySession we could call verifyTypedData using the same domain values that were used to sign the session

Reply:
would it make sense to leave it out of the EIP712 signer altogether? ideally we would not even ask the dev for a domain when they configure the signer

Reply:
closing this, i think it'll come up again during review/acceptance testing if it's an issue